<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmRMConfig</class>
 <widget class="QWidget" name="FrmRMConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1589</width>
    <height>1029</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget_wagon" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_11">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_12">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_13">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_14">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_15">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_16">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" rowspan="2">
       <widget class="QWidget" name="widget_function" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QToolButton" name="tbtn_measurePoint">
           <property name="minimumSize">
            <size>
             <width>160</width>
             <height>80</height>
            </size>
           </property>
           <property name="text">
            <string>测点配置</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="tbtn_bearing">
           <property name="minimumSize">
            <size>
             <width>160</width>
             <height>80</height>
            </size>
           </property>
           <property name="text">
            <string>轴承配置</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="tbtn_association">
           <property name="minimumSize">
            <size>
             <width>160</width>
             <height>80</height>
            </size>
           </property>
           <property name="text">
            <string>关联设备</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="tbtn_timertask">
           <property name="minimumSize">
            <size>
             <width>160</width>
             <height>80</height>
            </size>
           </property>
           <property name="text">
            <string>定时任务</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="tbtn_network">
           <property name="minimumSize">
            <size>
             <width>160</width>
             <height>80</height>
            </size>
           </property>
           <property name="text">
            <string>网络信息</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="tbtn_carinfo">
           <property name="minimumSize">
            <size>
             <width>160</width>
             <height>80</height>
            </size>
           </property>
           <property name="text">
            <string>车辆信息</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="tbtn_alarmlimit">
           <property name="minimumSize">
            <size>
             <width>160</width>
             <height>80</height>
            </size>
           </property>
           <property name="text">
            <string>报警阈值</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="tbtn_datasave">
           <property name="minimumSize">
            <size>
             <width>160</width>
             <height>80</height>
            </size>
           </property>
           <property name="text">
            <string>数据保存</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="tbtn_APPUpdate">
           <property name="minimumSize">
            <size>
             <width>160</width>
             <height>80</height>
            </size>
           </property>
           <property name="text">
            <string>设备升级</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="tbtn_devicereboot">
           <property name="minimumSize">
            <size>
             <width>160</width>
             <height>80</height>
            </size>
           </property>
           <property name="text">
            <string>设备重启</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="currentIndex">
         <number>-1</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
